performance-tests/performance.test.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-await-in-loop */
+// ^ disabled because interactions with the page can't be done in parallel
+
+const { getByText, getByLabelText, waitFor } = require('@testing-library/dom')
+const { default: userEvent } = require('@testing-library/user-event')
+const { startApp } = require('./test-app')
+
+/**
+ * This function helps test the element-renderer page, by setting the count, and hitting the render button
+ * @returns how long it took to render the elements
+ */
+const testElementRenderer = async (container, count, renders) => {
+	// setup {count} elements
+	userEvent.type(getByLabelText(container, 'Element Count'), '{selectall}{backspace}')
+	userEvent.type(getByLabelText(container, 'Element Count'), `${count}`)
+
+	// verify the count is {count}
+	await waitFor(() => {
+		expect(getByLabelText(container, 'Element Count')).toHaveValue(`${count}`)
+	})
+
+	// click to trigger the render
+	userEvent.click(getByText(container, 'Render'))
+
+	// wait for the render button data to udpate
+	await waitFor(() => {
+		expect(getByText(container, 'Render')).toHaveAttribute('renders', `${renders}`)
+	})
+
+	// will be "Wait: NN.NNNNNNNN"
```

Comment:
What does NN.NNNNNN mean? Is this a time or something else?

Reply:
this is supposed to represent what the text we are expecting... Not sure if there's a better way to word this...  In reality it'll look something like `Wait: 12.3456789`, which is maybe just what I should write 😆